src/pages/schedule-demo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Layout from '../components/Layout'
+import { DemoScheduler } from '../components/DemoScheduler'
+import { Spacer } from '../components/Spacer'
+import './styles/yc-onboarding.scss'
+
+export const ScheduleDemo = () => {
+    return (
+        <Layout>
+            <div className="yc-onboarding-wrapper">
+                <Spacer />
+                <h1 className="centered">Schedule Demo</h1>
+                <p>
+                    Use the widget below to schedule a demo call with Yakko, one of our engineers. During this slot we
```

Comment:
1. I'd make this wording a little more formal - it tends to be very large companies that book this way, so we need to not come across like a 2 person startup.

"Use the widget below to schedule a demo call with one of our engineers. We will discuss what you need, show you the product, get you set up if possible, and can give access to a demo instance to review after the call."

2. Doing the set up for these, which are often paid enquiries, is pretty cool. Curious to see how you get on asking people "look if you've time we can extend this by 30 mins and can get you set up right now, you only pay per event anyway so there's no minimum commitment". I'd probably then ensure you setup calendly to give you space to overrun if needed.

Reply:
1. Makes sense, one correction on your copy: "product" -> "platform" ;)

Reply:
2. Will configure this